docs/Set-PSReadLineOption.md

```diff
@@ -612,6 +615,28 @@ Accept pipeline input: false
 Accept wildcard characters: false
 ```
 
+### -PredictionStyle
+
+Specifies how PSReadLine should render the predictive suggestions.
+
+Valid values are:
+
+-- None: disable the predictive suggestion feature
+
+-- Concise: render the predictive suggestion in a concise view
+
+```yaml
+Type: PredictionStyle
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: Concise
+Accept pipeline input: false
```

Comment:
I really think this is a bad default.

Reply:
I'm fine to disable the prediction by default. Will update the code.